tests/director/tests/main_test.go

```diff
@@ -47,6 +47,7 @@ type DirectorConfig struct {
 	ConsumerSubaccountLabelKey             string
 	SubscriptionLabelKey                   string
 	RuntimeTypeLabelKey                    string
+	ApplicationTypeLabelKey                string `envconfig:"APP_APPLICATION_TYPE_LABEL_KEY,default=applicationType"`
```

Comment:
Are these explicit tags necessary? I see them in a couple of places within the pr.

Reply:
Not really necessary, but it's easier for me to check where the value comes from when searching, I'll change it to implicit if it's better that way.